content/teaching-curriculum/chapter1-high-standards/_index.md

```diff
@@ -2,32 +2,40 @@
 _db_id: 846
 content_type: project
 flavours:
-- none
+  - none
 prerequisites:
   hard:
-  - teaching-curriculum/quick-familiarity-1
+    - teaching-curriculum/quick-familiarity-1
+protect_main_branch: false
 ready: true
-submission_type: link
+submission_type: repo
 tags:
-- teaching
+  - teaching
 title: Chapter 1 - High Standards
 ---
 
-Here is the [12 page book summary](https://drive.google.com/file/d/1ace5039zhdNbrd4CBgXz3GikFpPwMLru/view?usp=share_link) which you'd have seen in the quick familiarity project(s). It might be useful if you need a very brief refresher on the various techniques. Note that the scope of focus for this whole course encompasses everything that we do as teachers at Umuzi, including Code Reviews, Technical Sessions, learner support and any other interactions we might have with learners. 
+Here is the [12-page book summary](https://drive.google.com/file/d/1ace5039zhdNbrd4CBgXz3GikFpPwMLru/view?usp=share_link) which you'd have seen in the quick familiarity project(s). It might be useful if you need a very brief refresher on the various techniques. Note that the scope of focus for this whole course encompasses everything that we do as teachers at Umuzi, including Code Reviews, Technical Sessions, learner support, and any other interactions we might have with learners.
 
-In this project, we're going to hone in on the first chapter, which is focused on techniques that set and uphold high academic standards. 
+In this project, we're going to hone in on the first chapter, which is focused on techniques that set and uphold high academic standards.
+
+These are the 5 techniques covered in the chapter:
 
-These are the 5 techniques covered in the chapter: 
 1. No Opt Out
 2. Right is right
 3. Stretch it
 4. Format matters
 5. Without apology
 
-## Your Mission Objectives
-1. For each of these 5 techniques, reflect on your own experiences as a learner (before Umuzi), and describe a teacher you can remember who applied the technique really well, or really poorly in your classroom. 
-2. Now reflect on your time at Umuzi, both as a learner and a teacher. For each of these 5 techniques, describe how/where you've seen it applied really well. In a separate paragraph, describe how/where you've seen it done really poorly. 
-3. What 1 system/change do you think Umuzi should put in place to improve how we uphold high academic standards? Describe your suggested system/change in at least 3, but no more than 6 short paragraphs. 
+## Your Mission Objectives:
```

Comment:
We usually don't use colons in headings I think. Headings already look like headings without colons.

If you think this is a better standard to follow then it would be good to get all the other content to follow this new standard.